### ISSUE TITLE
[iOS] Frame background color does not match the color of other elements

### ISSUE THREAD

**Issue by robinmanuelthiel** [xamarin/Xamarin.Forms#2218]
### Description

In Xamarin Forms on iOS, the `Frame` control background color does not match the color of other elements due to differences between CGColor and UIColor. @GalaxiaGuy has created a nice [GitHub Repository](https://github.com/GalaxiaGuy/xamarin-forms-frame-color) to reproduce this easily

### Steps to Reproduce

1. Create a `Page` with Background Color `#37404A`
2. Add a `Frame`with Background Color `#37404A`
3. See that `Frame`'s Background Color renderes differently than the `Page`s on iOS

### Expected Behavior

Both colors should be the same.

### Actual Behavior

Both colors are defined as the same but are rendered differently.

### Basic Information

- Version with issue: Xamarin.Forms 5.0.5280555

Most iOS renderers work using UIKit and UIColor. The background color for instance is set using UIView.BackgroundColor.

Frame however works using CoreGraphics and CGColor and manipulating a view's layer. The background color is set using UIView.Layer.BackgroundColor.

The bottom line, for consistency, `ToCGColor` should be defined as follows:

```csarp
public static CGColor ToCGColor(this Color color)
{
  return color.ToUIColor().CGColor;
}
```

### Screenshots

![Screenshot](https://github.com/GalaxiaGuy/xamarin-forms-frame-color/blob/master/screenshot.png?raw=true)

### Reproduction Link

https://github.com/GalaxiaGuy/xamarin-forms-frame-color

**Comment by jassmith** [comment]
One sets the Layer color, the other sets the UIView color, strange that this acts differently. Not sure we can "change" this now without an option but I am intrigued to dig in and see why this is different behaving since we are passing the same color to iOS.

**Comment by GalaxiaGuy** [comment]
Miguel made a blog post about it:

http://tirania.org/blog/archive/2017/Dec-07.html

**Comment by robinmanuelthiel** [comment]
Yeah, it's basically a problem of Xamarin.Forms `ToGCColor()` implementation. Both the ToCGColor and ToUIColor methods simply take the red, green, blue, and alpha components of the Xamarin Color directly. iOS is a bit more sophisticated than that, in that it supports different color spaces, making the conversion between CGColor and UIColor more complicated.

This describes it (and its solution) quite good: https://github.com/GalaxiaGuy/xamarin-forms-frame-color

**Comment by GalaxiaGuy** [comment]
It looks like this was fixed here: https://github.com/xamarin/Xamarin.Forms/pull/1182

**Comment by samhouts** [comment]
This issue doesn't seem to have had any activity in a long time. We're working on prioritizing issues and resolving them as quickly as we can. To help us get through the list, we would appreciate an update from you to let us know if this is still affecting you on the latest version of Xamarin.Forms, since it's possible that we may have resolved this as part of another related or duplicate issue. If we don't see any new activity on this issue in the next 30 days, we'll evaluate whether this issue should be closed. Thank you!

**Comment by samhouts** [comment]
Since we haven't heard from you in more than 30 days, we hope this issue is no longer affecting you. If it is, please reopen this issue and provide the requested information so that we can look into it further. Thank you!
